Client/src/Components/Inputs/Checkbox/index.jsx

```diff
@@ -65,7 +65,7 @@ const Checkbox = ({
 			sx={{
 				borderRadius: theme.shape.borderRadius,
 				p: theme.spacing(2.5),
-				m: theme.spacing(-2.5),
+				//m: theme.spacing(-2.5),
```

Comment:
Is this commented out temporarily?

If it's a permanent change then it should be removed not just commented out.

Reply:
Yes, I will remove that since it is a fix

Reply:
resolved in : https://github.com/bluewave-labs/bluewave-uptime/pull/1149/commits/a9aece6123eaa20ce2173c62c7f7904379a9f0dd
https://github.com/bluewave-labs/bluewave-uptime/pull/1149/commits/4e60981b0b1c487182b830991c89c117f6670006 etc